src/app/(not-authenticated)/sign_in/sign_in.tsx

```diff
@@ -2,10 +2,10 @@
 
 import { useState } from 'react';
 import { useRouter } from 'next/navigation';
-import BMLink from '../../../components/BMLink';
-import logo from '../../../../public/Title.png';
-import Input from '../../../components/Input';
-import Button from '../../../components/Button';
+import BMLink from '~/components/BMLink';
+import logo from '~/../public/Title.png';
```

Comment:
is this correct `~/../`

Reply:
The public folder is outside the src folder which is what ~ is defined to be.